### PR TITLE
scripts: Add windows-build-tarball.ps1

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: scripts/windows-build-tarball.ps1 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This script creates a tarball of the VTK Python package build tree. It is
```
Comment (jcfr) [inline scripts/windows-build-tarball.ps1]: `This script creates an archive ...`

Comment (thewtex) [reply scripts/windows-build-tarball.ps1]: Done

File: scripts/windows-build-tarball.ps1 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This script creates an archive of the VTK Python package build tree. It is
+# downloaded by the external module build scripts and used to build their
+# Python package on GitHub CI services.
+
```
Comment (jcfr) [inline scripts/windows-build-tarball.ps1]: Should we also add a comment like:

```
... on GitHub CI services.

It was designed to be executed on overload.kitware build machine and is invoked in XXXX after
ZZZ.
```

Comment (thewtex) [reply scripts/windows-build-tarball.ps1]: Done.

Comment (jcfr) [reply scripts/windows-build-tarball.ps1]: Excellent. Thanks for taking the time to accommodate all these nitpicks :+1:

**Comment by jcfr** [comment]
An other nitpick would be to rename these script from `windows-build-tarball.ps1 ` to `windows-build-achive.ps1` ... but if for consistency you want to keep `-tarball` prefix, that also works

**Comment by jcfr** [comment]
Or even better : `windows-generate-build-tree-achive.ps1`

**Pull request merged**